proof/crefine/AARCH64/Retype_C.thy

```diff
@@ -2215,7 +2215,9 @@ proof (intro impI allI)
     using al[simplified bit_simps]
     apply -
     apply (rule is_aligned_c_guard[where n="ptBits pt_t" and m=3])
-    apply (simp_all add:  align_td_array align_of_def bit_simps ptr0 split: if_split)
+    apply (simp_all add:  align_td_array align_of_def bit_simps ptr0 pt_t_def
+                          Kernel_Config.config_ARM_PA_SIZE_BITS_40_def
```

Comment:
I get a strange feeling when I see these unfoldings without a comment, and without a `solves`. For this one the `solves` is implicit as it's at the end of the proof, but the next one isn't. Thoughts on making these unfoldings better documented?

Reply:
Forcing a `solves` where that is not yet implied is probably a good idea, will look into that. It is a bit of a strange way to get genericity, but there are only two cases which we both test (just not in the same proof).